packages/plugin/src/index.ts

```diff
@@ -372,9 +372,13 @@ export default function typedocApiPlugin(
 				return {};
 			}
 
-			// Whitelist the folders that this webpack rule applies to,
-			// otherwise we collide with the native docs/blog plugins.
-			const include = packageConfigs.map((cfg) => path.join(options.projectRoot, cfg.packagePath));
+			// Whitelist the folders that this webpack rule applies to, otherwise we collide with the native
+			// docs/blog plugins. We need to include the specific files only, as in polyrepo mode, the `cfg.packagePath`
+			// can be project root (where the regular docs are too).
+			const include = packageConfigs.flatMap((cfg) => [
```

Comment:
alternatively we could whitelist the filenames on https://github.com/milesj/docusaurus-plugin-typedoc-api/pull/83/files#diff-a6a085138505e7ad124af8df6353efa6ec96cd337a74d0a522070bb2e4af0c53R387